gen.go

```diff
@@ -46,12 +75,30 @@ func doTemplate(w io.Writer, info interface{}, templ string) error {
 			"Deref": func(sp *string) string {
 				return *sp
 			},
+			"Cfg": func(prop string) string {
```

Comment:
What if we just modified `MaxLen`? I.e.:

1. Pass the var name (no `cbg.`) to `MaxLen`.
2. Put this switch inside `MaxLen`.

Reply:
fair enough, it makes the `ReadString` slightly awkward but does simplify the rest; I've also shortened the names for that case too; see latest commit.